src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import './PostList.scss';
+import { PostInfo } from '../PostInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  post: Todo[];
+};
+
+export const PostList: React.FC<Props> = ({ post }) => (
+  <div className="PostList">
+
+    { post.map(content => (
+      content.user
+      && (
+        <PostInfo {...content} />
```

Comment:
don't pass props in this way
you should define one object in props and pass entire object